iroh-net/src/magicsock/peer_map/endpoint.rs

```diff
@@ -108,10 +102,10 @@ pub struct PendingCliPing {
 
 #[derive(Debug)]
 pub(super) struct Options {
-    pub(super) public_key: PublicKey,
-    pub(super) derp_region: Option<u16>,
+    pub public_key: PublicKey,
+    pub derp_region: Option<u16>,
```

Comment:
why is this `pub` if the struct itself is `pub(super)`?

Reply:
changed to `pub(super)`